staging/src/k8s.io/controller-manager/config/v1alpha1/conversion.go

```diff
@@ -18,8 +18,10 @@ package v1alpha1
 
 import (
 	"k8s.io/apimachinery/pkg/conversion"
+	cpconfig "k8s.io/cloud-provider/config"
+	serviceconfig "k8s.io/cloud-provider/service/config"
```

Comment:
the v1alpha1 KubeCloudSharedConfiguration and ServiceControllerConfiguration types (and the conversions from this file) should move to `k8s.io/cloud-provider` to drop these dependencies